src/jobmaker/jobmaker.cfg

```diff
@@ -15,6 +15,9 @@ jobmaker = {
   # max gbt life cycle time seconds
   gbt_life_time = 60;
 
+  # max empty gbt life cycle time seconds
```

Comment:
# "empty_gbt_life_time" SHOULD >= 10, because we reduce the stratum jobs interval to 10 seconds (hard coding) when the latest stratum job is an empty block job.